cobra/tools/burnapp.py

```diff
@@ -1,38 +1,49 @@
 import os
 import sys
+import logging
 import zipfile
+import binascii
 import optparse
 import tempfile
 
 import cobra
 import cobra.remoteapp as c_remoteapp
+import envi.common as e_common
+
+logger = logging.getLogger(__name__)
+e_common.setLogging(logger, 'INFO')
```

Comment:
A lot of the logger stuff was a kind of blunt hammer conversion. I tried to keep parity where I could, and while I don't know who or what uses this, I figured I'd leave it in for now. since it seems like a more public facing tool.